mem.go

```diff
@@ -75,6 +75,17 @@ func (e *memEngine) DropCollection(name string) error {
 	return nil
 }
 
+func (e *memEngine) Count(q *Query) (int, error) {
+	var num int
```

Comment:
Just because it looks to me more clear, in both methods I would initialize num to 0 like so:
```golang
num := 0
```

Reply:
yes , u r right, initializing num to 0 make it easier to read